server/netty-server/src/main/scala/sttp/tapir/server/netty/internal/RunAsync.scala

```diff
@@ -1,5 +1,19 @@
 package sttp.tapir.server.netty.internal
 
+import scala.concurrent.Future
+
 trait RunAsync[F[_]] {
   def apply(f: => F[Unit]): Unit
 }
+object RunAsync {
+  type Id[A] = A
+  
+  final val Id: RunAsync[Id] = new RunAsync[Id] {
```

Comment:
out of curiosity: anything better in `final val Id` over `object Id extends RunAsync[Id]`?

Reply:
Hmm, no strong opinions.

It arguably generates somewhat simpler bytecode, i.e.
* no lazy initialization associated with `object`
* no separate class associated with `object` (there's an anonymous class but it will probably be compiled to a lambda so it won't have a classfile)

It also works more naturally with type inference:
* when it's a `val`, type of `Id` will be inferred as `RunAsync[Id]` (unless explicitly requested to be typed as `Id.type`)
* when it's an `object`, it will be inferred as `Id.type`, with API potentially extended over `RunAsync[Id]`

Neither the laziness nor introducing a subtype was my intention, so a `val` is much closer to expressing exactly what I want, which is just having a plain implementation of `RunAsync`.

Reply:
sounds good, thanks :)